content/docs/csidriver/installation/test/certcsi.md

```diff
@@ -341,6 +331,32 @@ To run block snapshot test suite, run the command:
 cert-csi test blocksnap --sc <storageClass> --vsc <snapshotclass>
 ```
 
+#### Volume Health Metric Suite
+
+To run volume health metric test suite, run the command:
+```bash
+
+cert-csi test volumehealthmetrics --sc <storage-class> --driver-ns <driver-namespace> --podNum <number-of-pods> --volNum <number-of-volumes>
+```
+
+> Note: Make sure to enable healthMonitor for driver's controller and node pods before running this suite. It is recommended to use a smaller interval time for this sidecar.
+
+#### Ephemeral volumes suite
+
+To run ephemeral volume test suite, run the command:
+```bash
+cert-csi test ephemeral-volume --driver <driver-name> --attr ephemeral-config.properties
+--pods : Number of pods to create 
```

Comment:
In a list like this, use a parallel structure for each item: 
-pods : Number of pods to create 
--pod-name : Custom name for the pod
--attr : File name for the CSI volume attributes file
--fs-type: FS Type (optional)